Make node_dfs optional in from_dfs integration (#171)
neo4j/python-graph-visualization: changelog.md

## New features

> * Allow visualization from only based relationship DataFrames without specifiying the nodes DataFrames in `from_df[s]`

Reviewer: ```suggestion
* Allow visualization based only on relationship DataFrames, without specifiying node DataFrames in `from_dfs`
```
